core/amber/src/main/python/core/runnables/network_receiver.py

```diff
@@ -59,10 +63,15 @@ def data_handler(command: bytes, table: Table) -> int:
             :return: sender credits
             """
             data_header = PythonDataHeader().parse(command)
+            marker = data_header.marker
+            if marker == "data":
```

Comment:
confusing. how can `marker == "data"`

Reply:
I will make the marker empty when it's data.